### PR TITLE
BaSD/MS-46/Postulants redux

### PR DESCRIPTION
**Pull request by MicaQuarleri**

### CODE REVIEW

**Comment by aquilesmariani** [comment]
@MicaQuarleri  Let's pause this one until we merge https://github.com/BaSD-2021/Aquiles-MindSet-app/pull/20 since it has a lot of changes that were made on that branch

**Comment by aquilesmariani** [comment]
@MicaQuarleri since you merged [your previous PR](https://github.com/BaSD-2021/Aquiles-MindSet-app/pull/20), you have conflicts to solve. Please merge development to this branch and then we can review this one.

File: src/Components/Councelors/index.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ const Councelor = () => {
   const error = useSelector((store) => store.counselors.error);
   const errorMessage = useSelector((store) => store.counselors.errorMessage);
 
+  console.log(counselors);
+
```
Comment (ignacioaristo) [inline src/Components/Councelors/index.js:26]: Remove the `console.log`

File: src/redux/postulants/constants.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const GET_POSTULANTS_FETCHING = 'GET_POSTULANTS_FETCHING';
+export const GET_POSTULANTS_FULFILLED = 'GET_POSTULANTS_FULFILLED';
+export const GET_POSTULANTS_REJECTED = 'GET_POSTULANTS_REJECTED';
+export const GET_ONE_POSTULANT_FETCHING = 'GET_ONE_POSTULANT_FETCHING'; //note singular here
+export const GET_ONE_POSTULANT_FULFILLED = 'GET_ONE_POSTULANT_FULFILLED';
```
Comment (ignacioaristo) [inline src/redux/postulants/constants.js:5]: Maybe remove this comments at the end of the line.

**Review by ignacioaristo — CHANGES_REQUESTED** [review]

**Review by ignacioaristo — APPROVED** [review]

**Review by aquilesmariani — APPROVED** [review]
Looks nice @MicaQuarleri!

**Pull request merged**